en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>response</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Response latency(95th percentile) in milliseconds(ms).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>backend</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Backend latency(95th percentile) in milliseconds(ms).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>requestMediation</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Request mediation latency(95th percentile) in milliseconds(ms).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>responseMediation</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Response mediation latency(95th percentile) in milliseconds(ms).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>responseMedian</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Median(50th percentile) of the response latency in milliseconds(ms).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>backendMedian</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Median(50th percentile) of the backend latency in milliseconds(ms).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>requestMediationMedian</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Median(50th percentile) of the request mediation latency in milliseconds(ms).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>responseMediationMedian</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Median(50th percentile) of the response mediation latency in milliseconds(ms).
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### LatencySummary
+
+Provides latency summary.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>summary</strong></td>
+<td valign="top">[<a href="#latencyvalue">LatencyValue</a>]!</td>
+<td>
+
+List containing latency values. Returns an empty array if no data available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Granularity value used for data retrieval.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### LatencyValue
+
+Represents latency value for a single timestamp. Timestamp is calculated according to the provided granularity value.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>timeSpan</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Represents a timestamp value calculated according to the used granularity value. For an example if the used
```

Comment:
When you say "used granularity value" it is not clear as to where it is used and by what. Please check whether the following suggestion gives the same meaning.

```suggestion
Represents a timestamp value that is calculated based on the specified granularity value. For example, if the granularity value is `1d`, the timestamp represents the start of the day calculated adhering to the timezone provided in the time filter.
e.g., timeSpan: '2021-06-21T00:00:00.0000000+05:30'
```